include/volk/volk_avx_intrinsics.h

```diff
@@ -125,4 +125,14 @@ _mm256_polar_fsign_add_llrs(__m256 src0, __m256 src1, __m128i fbits){
     return dst;
 }
 
+static inline __m256
+_mm256_accumulate_square_sum(__m256 sq_acc, __m256 acc, __m256 val, __m256 rec, __m256 aux)
```

Comment:
The function name should end with `_ps` like the corresponding intrinsics do. That'll make it more in line with those intrinsics.